inductiva/tests/benchmarks/test_benchmarks.py

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
Move the tests to `test_machines.py`

Reply:
Done!

Reply:
I believe we can merge since the code was already tested on the original PR. This PR was just a refactor (and to delete a method that I forgot to remove)